src/main/kotlin/ch/wisv/choice/document/model/Document.kt

```diff
@@ -1,11 +1,15 @@
 package ch.wisv.choice.document.model
 
-import ch.wisv.choice.exam.model.Exam
-import javax.persistence.*
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.Lob
 
 @Entity
 data class Document(
         @GeneratedValue @Id val id: Long? = null,
+
         @Lob var bytes: ByteArray,
-        var name: String,
-        @ManyToOne(targetEntity = Exam::class) var exam: Exam)
```

Comment:
Why are we decoupling exam and document?

Reply:
Because it is the other way around

Reply:
An exam can have multiple documents, questions/answer sheet. So no